src/layouts/Home/components/Foundation.tsx

```diff
@@ -81,31 +81,37 @@ const FoundationLink: React.FC<{
 export const Foundation: React.FC = () => {
   const { contentImage, componentsImage, designImage } = useStaticQuery(
     graphql`
-      query {
+      {
         contentImage: abstractAsset(layerName: { eq: "home-pillars-content" }) {
           childFile {
             childImageSharp {
-              fluid(maxWidth: 350, traceSVG: { background: "#F6F4F4", color: "#228F67" }) {
-                ...GatsbyImageSharpFluid_tracedSVG
-              }
+              gatsbyImageData(
```

Comment:
Here's a [documentation list](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#all-options) of all the options available for `gatsbyImageData`.